lib/rules/selector-max-compound-selectors/README.md

```diff
@@ -59,3 +59,43 @@ div {}
 ```css
 .foo + div :not (a b ~ c) {} /* `a b ~ c` is inside `:not()`, so it is evaluated separately */
 ```
+
+## Optional secondary options
+
+### `ignoreSelectors: ["/regex/", /regex/, "non-regex"]`
+
+Ignore some compound selectors. This may be useful for deep selectors like Vue's `::v-deep` or Angular's `::ng-deep` that behave more like combinators than compound selectors.
+
+For example, with `2`.
+
+Given:
+
+```json
+["::v-deep", "/ignored/"]
+```
+
+The following patterns are considered problems:
+
+<!-- prettier-ignore -->
+```css
+.foo .bar ::v-deep .baz {}
+```
+
+The following patterns are _not_ considered problems:
+
+<!-- prettier-ignore -->
+```css
+.foo::v-deep .bar {}
+```
+
+<!-- prettier-ignore -->
+```css
+.foo ::v-deep .baz {}
+```
+
+<!-- prettier-ignore -->
+```css
+.some {
+  &-ignored-class ::v-deep > .bar {}
```

Comment:
[suggest] `&-ignored-class` is non-standard CSS (e.g. SCSS), right? I think we should avoid non-standard syntaxes even in examples.

See also: https://github.com/stylelint/stylelint/blob/b6446093297c9a6a147c3c967067f06c12d76ab1/docs/developer-guide/rules.md?plain=1#L11-L13

Reply:
You're right, see https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_nesting/Using_CSS_nesting#concatenation_is_not_possible. I changed it in [`1df6e37`](https://github.com/stylelint/stylelint/pull/7544/commits/1df6e37fee0043b8da359035cd294d7307027b36).